.github/workflows/benchmark-auto-typed.yml

```diff
@@ -0,0 +1,37 @@
+name: TypeScript Benchmark "auto-typed-mongoose"
```

Comment:
Let's keep this in the `benchmark.yml` file to minimize overhead. Just add 2 extra steps to `npm install` and run ts-benchmark in auto-typed directory.